### DISCUSSION TITLE
Build with 11ty

### DISCUSSION THREAD

**Discussion by shawnthompson** [gc-da11yn/gc-da11yn.github.io#2] [Development (Open discussions)]
Build the site based on what the [a11yproject](https://github.com/a11yproject/a11yproject.com) is doing.

It uses [Eleventy](https://www.11ty.io), a simple static site generator using [nodeJS](https://nodejs.org/en/).

Deploying with [Github Actions](https://github.com/features/actions) to [Github Pages](https://pages.github.com) to host the site.

**Comment by sboots** [comment]
@shawnthompson This looks brilliant, super nice work! Great choice of tech stack too. 🙌  🙌

**Reply by shawnthompson** [reply]
Thank you very much @sboots

**Comment by shawnthompson** [comment]
A great video on 11ty. : [Let’s Learn Eleventy! (with Zach Leatherman) — Learn With Jason](https://youtu.be/j8mJrhhdHWc)
